### ISSUE TITLE
Running with Java SE 11.0.1

### ISSUE THREAD

**Issue by srigurubyo** [sismics/docs#258]
Anyone has tried running sismics on latest Java SE 11.0.1 version?  Would appreciate if you could some lights on this.

**Comment by srigurubyo** [comment]
I could successfully run on Java SE 11.  One change I had to do it to manually add jaxb dependencies as they are remove from version 11.

  ```
  <dependency>
        <groupId>javax.xml.bind</groupId>
        <artifactId>jaxb-api</artifactId>
        <version>2.3.0</version>
    </dependency>
    <dependency>
        <groupId>com.sun.xml.bind</groupId>
        <artifactId>jaxb-impl</artifactId>
        <version>2.3.0</version>
    </dependency>
    <dependency>
        <groupId>org.glassfish.jaxb</groupId>
        <artifactId>jaxb-runtime</artifactId>
        <version>2.3.0</version>
    </dependency>
    <dependency>
        <groupId>javax.activation</groupId>
        <artifactId>activation</artifactId>
        <version>1.1.1</version>
    </dependency>  
```

**Comment by jendib** [comment]
That's strange because we don't use jaxb. Maybe a hard dependency from Jersey.
I'm going to investigate this.

**Comment by srigurubyo** [comment]
Please let me know if there is anything I can help here.  Glad to do.

**Comment by jendib** [comment]
Closed by #287
